// Catch Condicional

try {
    adsadadssadsa
} 

// "catch" condicional: presenta una condicion dentro
catch (error) 
{   
    if (3 < 5) 
    {
        console.log("Ha ocurrido un error");
    }
    else
    {
        console.log("pito");
    }
}